google/cloud/opentelemetry/internal/recordable.cc

```diff
@@ -28,6 +28,29 @@ namespace {
 // http://35.193.25.4/TensorFlow/models/research/syntaxnet/util/utf8/unilib_utf8_utils.h
 bool IsTrailByte(char x) { return static_cast<signed char>(x) < -0x40; }
 
+// OpenTelemetry's semantic conventions for attribute keys differ from Cloud
+// Trace's semantics for label keys. So translate from one to the other.
+//
+// See: https://cloud.google.com/trace/docs/trace-labels#canonical_labels
+opentelemetry::nostd::string_view MapKey(
+    opentelemetry::nostd::string_view key) {
+  static auto const* m = new std::map<std::string, std::string>{
```

Comment:
It seems like you should be able to make this `string_view -> string_view` and avoid the conversion to `string` on lookup.

Reply:
Fixed, thanks.